src/internet_identity/src/anchor_management.rs

```diff
@@ -51,20 +51,27 @@ pub fn get_anchor_info(anchor_number: AnchorNumber) -> IdentityAnchorInfo {
     })
 }
 
+/// Handles all the bookkeeping required on anchor activity:
+/// * Records anchor activity within aggregated activity stats
+/// * Updates the last usage timestamp
+///
+/// Note: modifies the anchor but not does not write to storage.
```

Comment:
nit: maybe add more explanation of why this is the case (as we had it for `update_last_device_usage`) -- otherwise it is not clear why this exceptional treatment.